### ISSUE TITLE
Add a Chatbot to the App

### ISSUE THREAD

**Issue by prajjwaldimri** [prajjwaldimri/Friend-App#53]
Could be useful for in-app support and fun chats (for depressed people also)

https://dev.botframework.com/

**Comment by RohanRon** [comment]
yeah this is an idea though we'll have to ponder whether it will actually
be useful since the app is designed for SOS / emergency
